### PR TITLE
feat: add additional fields for `Employee` and related types

### PR DESCRIPTION
**Pull request by svetavedmed**
https://github.com/edenlabllc/ehealth.api/issues/4483

### CODE REVIEW

File: __schema__/employees.graphql [hunk 1]
```diff
@@ -147,3 +163,75 @@ type Speciality {
...
+  "The date when science degree was obtained."
+  issuedDate: Date
+  "Speciality that in mentioned in science degree certificate"
+  speciality: String!
+}
+
+"""
+Input fields to `DeactivateEmployee`. User must have scope `employee:deactivate`
+"""
+input DeactivateEmployeeInput {
```
Comment (kiurchv) [inline __schema__/employees.graphql]: Mutation input and payload types should go before a node types

File: __schema__/index.graphql [hunk 2]
```diff
@@ -163,6 +163,11 @@ type Query {
   "Reads a single `Employee` using its globally unique ID."
   employee(id: ID!): Employee
 
+  """
+  Deactivate `Employee` with given field values.
+  """
+  deactivateEmployee(input: DeactivateEmployeeInput!): DeactivateEmployeePayload
```
Comment (kiurchv) [inline __schema__/index.graphql]: This field should be in the `Mutation` type

File: __schema__/employees.graphql [hunk 3]
```diff
@@ -147,3 +163,75 @@ type Speciality {
...
+  speciality: String!
+}
+
+"""
+Information about employee's qualification
+"""
+type Qualification {
+  "Number of qualification certificate."
+  certificateNumber: String
+  "Oficial name of institution"
```
Comment (ephor) [inline __schema__/employees.graphql]:
```suggestion
  "Official name of institution"
```

File: __schema__/employees.graphql [hunk 4]
```diff
@@ -147,3 +163,75 @@ type Speciality {
...
+type ScienceDegree {
+  "The city where science degree was obtained"
+  city: String!
+  "The country where science degree was obtained"
+  country: String!
+  "Science degree"
+  degree: String!
+  "Name science degree diploma"
+  diplomaNumber: String!
+  "Oficial name of institution"
```
Comment (ephor) [inline __schema__/employees.graphql]:
```suggestion
  "Official name of institution"
```

File: __schema__/employees.graphql [hunk 5]
```diff
@@ -147,3 +163,75 @@ type Speciality {
...
+  issuedDate: Date
+  "Speciality that in mentioned in science degree certificate"
+  speciality: String!
+}
+
+"""
+Input fields to `DeactivateEmployee`. User must have scope `employee:deactivate`
+"""
+input DeactivateEmployeeInput {
+  "Input for a single `contractId` using its globally unique ID."
```
Comment (kiurchv) [inline __schema__/employees.graphql]:
```suggestion
  "Globally unique ID of the `Employee` which should be deactivated."
```

File: __schema__/parties.graphql [hunk 6]
```diff
@@ -21,7 +21,19 @@ type Party {
   "Employee's gender, is choosen from dictionary."
   gender: Gender!
   "Personal phone number."
-  phones: [Phone]
+  phones: [Phone]!
```
Comment (ephor) [inline __schema__/parties.graphql]: Why this field become required?

File: __schema__/parties.graphql [hunk 7]
```diff
@@ -21,7 +21,19 @@ type Party {
...
+  "National person identifier."
+  taxId: String!
+  "Flag to show whether person rejected to have taxId"
+  noTaxId: Boolean
+  "Party identification documents."
+  documents: [PartyDocument]!
+  "Party's email"
+  email: String!
+  "Introdaction about party"
```
Comment (ephor) [inline __schema__/parties.graphql]:
```suggestion
  "Party self-description"
```

File: __schema__/parties.graphql [hunk 8]
```diff
@@ -21,7 +21,19 @@ type Party {
...
+  aboutMyself: String
+  "Experience in working"
```
Comment (kiurchv) [inline __schema__/parties.graphql]:
```suggestion
  "Amount of time during which a party holds its career, in years."
```

File: __schema__/parties.graphql [hunk 9]
```diff
@@ -33,3 +45,17 @@ enum Gender {
...
+Party identifier documents.
+"""
+type PartyDocument {
+  "Dictionary DOCUMENT_TYPE"
+  type: String!
+  "document issue number"
+  number: String!
+  "authority which issued the document"
+  issuedBy: String @fake(locale: "uk", method: "company.companyName")
+  "the date when document was issued"
```
Comment (kiurchv) [inline __schema__/parties.graphql]:
```suggestion
  "The date when the document was issued."
```

File: __schema__/parties.graphql [hunk 10]
```diff
@@ -33,3 +45,17 @@ enum Gender {
...
+
+"""
+Party identifier documents.
+"""
+type PartyDocument {
+  "Dictionary DOCUMENT_TYPE"
+  type: String!
+  "document issue number"
+  number: String!
+  "authority which issued the document"
```
Comment (kiurchv) [inline __schema__/parties.graphql]:
```suggestion
  "Authority which issued the document."
```

File: __schema__/parties.graphql [hunk 11]
```diff
@@ -33,3 +45,17 @@ enum Gender {
...
   MALE
 }
+
+"""
+Party identifier documents.
+"""
+type PartyDocument {
+  "Dictionary DOCUMENT_TYPE"
+  type: String!
+  "document issue number"
```
Comment (kiurchv) [inline __schema__/parties.graphql]:
```suggestion
  "An issue number of the document."
```

File: __schema__/parties.graphql [hunk 12]
```diff
@@ -33,3 +45,17 @@ enum Gender {
   "Gender `MALE` for a party."
   MALE
 }
+
+"""
+Party identifier documents.
+"""
+type PartyDocument {
+  "Dictionary DOCUMENT_TYPE"
```
Comment (kiurchv) [inline __schema__/parties.graphql]:
```suggestion
  "Document type. Value should be present in the `DOCUMENT_TYPE` dictionary."
```

File: __schema__/employees.graphql [hunk 13]
```diff
@@ -54,6 +64,14 @@ type EmployeeEdge {
   cursor: String!
 }
 
+"""
+Input fields to `DeactivateEmployee`. User must have scope `employee:deactivate`
+"""
+input DeactivateEmployeeInput {
+  "Input for a single `contractId` using its globally unique ID."
```
Comment (kiurchv) [inline __schema__/employees.graphql]:
```suggestion
  "Globally unique ID of the `Employee` which should be deactivated."
```

File: __schema__/index.graphql [hunk 14]
```diff
@@ -541,6 +541,13 @@ type Mutation {
   deactivateMedication(
     input: DeactivateMedicationInput!
   ): DeactivateMedicationPayload
+  
+  """
+  Deactivate `Employee` with given field values.
```
Comment (kiurchv) [inline __schema__/index.graphql]:
```suggestion
  Deactivates a single `Employee` using its globally unique ID.
```

File: __schema__/employees.graphql [hunk 15]
```diff
@@ -54,6 +64,14 @@ type EmployeeEdge {
...
+  id: ID!
+}
+
 """
 Employee details contain information about Party (personal data) and Employee (proffesional data)
```
Comment (kiurchv) [inline __schema__/employees.graphql]:
```suggestion
Employee details contain information about Party (personal data) and Employee (professional data)
```

File: __schema__/parties.graphql (hunk 6 above)
Comment (svetavedmed) [reply __schema__/parties.graphql]: it is required on [json schemas](https://github.com/edenlabllc/ehealth.api/blob/develop/apps/core/specs/json_schemas/employee_request/employee_request_create_request.json)

**Pull request merged**